pkg/operator/README.md

```diff
@@ -40,6 +40,16 @@ post-install configurations should probably move here.
 * monitor and repair mdsd as needed
 * set the alertmanager webhook
 
+### Controllers and Deployment
+
+The full list of operator controllers with descriptions can be
+found in the README at the root of the repository.
+
+The static pod resources can be found at `pkg/operator/deploy/staticresources`. The
+deploy operation kicks off two deployments in the `openshift-azure-operator` namespace, one for
+master and one for worker. The `aro-operator-master` deployment runs all controllers,
+while the `aro-operator-worker` deployment runs only the internet and service principal checkers in the worker subnet.
```

Comment:
The worker runs the `internetchecker` because on line 35 of `pkg/operator/controllers/checker/checker_controller.go` you can see that if it's roleMaster that it will append that to the list of checkers on the master deployment

Reply:
Done, nice catch.